consensus/core/src/network/tonic_network.rs

```diff
@@ -775,18 +775,36 @@ impl<S: NetworkService> NetworkManager<S> for TonicManager {
             }
         }
 
-        let server = sui_http::Builder::new()
-            .config(
-                sui_http::Config::default()
-                    .initial_connection_window_size(64 << 20)
-                    .initial_stream_window_size(32 << 20)
-                    .http2_keepalive_interval(Some(config.keepalive_interval))
-                    .http2_keepalive_timeout(Some(config.keepalive_interval))
-                    .accept_http1(false),
-            )
-            .tls_config(tls_server_config)
-            .serve(own_address, consensus_service)
-            .unwrap();
+        let http_config = sui_http::Config::default()
+            .initial_connection_window_size(64 << 20)
+            .initial_stream_window_size(32 << 20)
+            .http2_keepalive_interval(Some(config.keepalive_interval))
+            .http2_keepalive_timeout(Some(config.keepalive_interval))
+            .accept_http1(false);
+
+        // Create server
+        //
+        // In the event of a crash or restart (say in simtest) there may be an older instance of
+        // consensus running that is bound to the TCP port of `own_address` that hasn't finished
```

Comment:
I don't think this is true outside of simtest.

Reply:
so this is specifically for simtest then, happy to update the comment